### DISCUSSION TITLE
Add a confirmation box when deleting requests

### DISCUSSION THREAD

**Discussion by DimitriosKozomara** [thunderclient/thunder-client-support#1487] [Ideas]
When creating tests in Thunder-Client, I often duplicate previous requests and modify them for new test cases. However, I frequently encounter an issue where I accidentally press the 'Delete' button instead of the 'Duplicate' option.

I would greatly appreciate it if Thunder-Client could include a confirmation box for the 'Delete' option, or even better, a configurable setting to prevent accidental deletions.

**Comment by rangav** [comment]
Thanks @DimitriosKozomara for the feedback, Will add the delete confirmation prompt in the next update.

**Reply by DimitriosKozomara** [reply]
Thank you!

**Comment by rangav** [comment]
This feature has been implemented and published to the marketplace. Please update to `v2.19.0`.

You can see all the features released in this update
https://github.com/rangav/thunder-client-support/releases/tag/v2.19.0

Please test it and let me know your feedback

**Reply by DimitriosKozomara** [reply]
Thank you for implementing the change I've requested! I've tested the new feature, and it's working flawlessly. That's exactly what I've wanted.

**Reply by rangav** [reply]
Thanks for the confirmation
